microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Using the Microsoft Edge Add-ons API (under development)
+title: Using the Microsoft Edge Add-ons API (in private preview)
 description: REST endpoints to automate publishing updates to add-ons that are submitted to the Microsoft Edge Add-ons website.
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.date: 08/19/2021
 ---
-# Using the Microsoft Edge Add-ons API (under development)
+# Using the Microsoft Edge Add-ons API (in private preview)
 
 > [!NOTE]
-> This article is a Request for Comments.  The Microsoft Edge Add-ons API is not yet available for testing, and the Publish APIs page is not yet available at Partner Center.  The Microsoft Edge Add-ons API is under active development and the roadmap continues to evolve based on market changes and customer feedback.  The plans outlined here are not exhaustive and are subject to change.
+> The Microsoft Edge Add-ons API is currently in private preview, and the Publish APIs page is not yet available at Partner Center for everyone.  The Microsoft Edge Add-ons API is under active development and the roadmap continues to evolve based on market changes and customer feedback.  The plans outlined here are not exhaustive and are subject to change.
```

Comment:
The page title at Partner Center is **Publish API** not **Publish APIs**; see screenshot at https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/using-addons-api?branch=pr-en-us-1694#before-you-begin
```suggestion
> The Microsoft Edge Add-ons API is currently in private preview.  The Publish API page is available at Partner Center only for participants of the private preview.  The Microsoft Edge Add-ons API is under active development and the roadmap continues to evolve based on market changes and customer feedback.  The plans outlined here aren't exhaustive, and are subject to change.
```